app/views/session.py

```diff
@@ -24,14 +39,14 @@ def export_session(**kwargs):
 def import_session(**kwargs):
     responses = {}
     data = request.get_json()
-
+    clear_session_data()
     for user_data in data['item']:
         try:
-            item = Item.objects.get(unique=user_data['unique'])
+            item = Item.objects.get(_id=user_data['_id'])
             item.available = user_data['available']
             item.claimed = user_data['claimed']
             item.save()
-            responses.add('200')
+            responses.add('204')
         except DoesNotExist:
             responses.add('400')
 
```

Comment:
Might as well skip trying users if items failed

Reply:
I'll just have it early return on failure